figure_test.go

```diff
@@ -0,0 +1,16 @@
+package figure
+
+import (
+	"fmt"
+)
+
+func ExampleAlphabet() {
+	myFigure := NewFigure("Hello world", "alphabet", true)
```

Comment:
Would you mind adding a test to ensure the default font works?  `myFigure := NewFigure("Hello world", "", true)`.